server/datastore/mysql/software_installers.go

```diff
@@ -291,8 +300,7 @@ SELECT
 		ELSE NULL -- not installed from Fleet installer
 	END, '') AS status,
 	si.filename AS software_package,
-	h.team_id AS host_team_id,
-	hsi.user_id AS user_id
```

Comment:
is this intentional?

Reply:
No, looks like a conflict resolution fatfinger on my part.